tools-local/tasks/mobile.tasks/CreateWasmBundle/DownloadTimeZoneData.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Net;
+using System.Reflection;
+using System.Diagnostics;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+public class DownloadTimeZoneData : Task
+{
+    [Required]
+    public string? InputDirectory { get; set; }
+
+    [Required]
+    public string? OutputDirectory { get; set; }
+
+    [Required]
+    public string? Version { get; set; }
+
+    private void DownloadTimeZoneDataSource() 
+    {
+        List<string> files = new List<string>() {"africa", "antarctica", "asia", "australasia", "etcetera", "europe", "northamerica", "southamerica", "zone1970.tab"};
+        using (var client = new WebClient())
+        {
+            foreach (var file in files) 
+            {
+                client.DownloadFile($"https://data.iana.org/time-zones/tzdb-{Version}/{file}", $"{Path.Combine(InputDirectory!, file)}");
```

Comment:
It might be helpful to print what it's doing in case the download takes long time